server/utils/applicationUtils.ts

```diff
@@ -176,12 +185,11 @@ const taskResponsesToSummaryListRowItems = (
                 ? formatLines(value as string)
                 : embeddedSummaryListItem(value as Array<Record<string, unknown>>),
           },
+          actions: actionItems ? { items: actionItems } : undefined,
```

Comment:
should this be checking for `actionItems.length`?

Reply:
Nope, `actionItems` is either an array of `SummaryListActionItem` or undefined -- we only need to set those if defined.